typescript/infra/config/environments/testnet4/agent.ts

```diff
@@ -368,7 +374,7 @@ const hyperlane: RootAgentConfig = {
     rpcConsensusType: RpcConsensusType.Fallback,
     docker: {
       repo,
-      tag: '385b307-20250418-150728',
+      tag: '9bf1f69-20250507-100731',
```

Comment:
`hyperlane` instead of `rc` whilst starknet works on migrating to `neutron` context